libpod/container_internal.go

```diff
@@ -786,53 +781,6 @@ func (c *Container) mountStorage() (err error) {
 	return c.save()
 }
 
-// prepare mounts the container and sets up other required resources like net
-// namespaces
-func (c *Container) prepare() (err error) {
```

Comment:
Again, why are we removing the entirety of these functions? I think all we really need here is to move the `c.config.CreateNetNS && c.state.NetNS == nil && c.config.PostConfigureNetNS` check into `createNetNS()` (and just have it return nil if that doesn't pass)

Reply:
per our discussion, whoever does darwin|windows needs to parse this out